webpack/assets/javascripts/react_app/routes/common/PageLayout/PageLayout.js

```diff
@@ -20,47 +25,52 @@ const PageLayout = ({
   isLoading,
   children,
 }) => (
-  <div id="main">
-    <div id="react-content">
-      <Head>
-        <title>{header}</title>
-      </Head>
-      <div id="breadcrumb">
-        {!breadcrumbOptions && (
-          <div className="row form-group">
-            <h1 className="col-md-8">{header}</h1>
-          </div>
-        )}
-        {customBreadcrumbs ||
-          (breadcrumbOptions && <BreadcrumbBar {...breadcrumbOptions} />)}
-      </div>
-      {beforeToolbarComponent}
-      <Row>
-        <Col className="title_filter" md={searchable ? 6 : 4}>
-          {searchable && (
-            <SearchBar
-              data={{
-                ...searchProps,
-                autocomplete: { ...searchProps.autocomplete, searchQuery },
-              }}
-              onSearch={onSearch}
-            />
-          )}
-          &nbsp;
-        </Col>
-        <Col id="title_action" md={searchable ? 6 : 8}>
-          <div className="btn-toolbar pull-right">
-            {isLoading && (
-              <div id="toolbar-spinner">
-                <Spinner loading size="sm" />
-              </div>
-            )}
-            {toolbarButtons}
-          </div>
-        </Col>
-      </Row>
-      {children}
+  <div id="react-content">
+    <Head>
+      <title>{header}</title>
+    </Head>
+    <div id="breadcrumb">
+      {!breadcrumbOptions && (
+        <PageSection variant={PageSectionVariants.light}>
```

Comment:
move `<PageSection` to be above the `<div id="breadcrumb">`
and remove `PageSection` from `Breadcrumb`